projects/control-service/projects/pipelines_control_service/src/integration-test/resources/application-test.properties

```diff
@@ -43,22 +49,19 @@ spring.security.oauth2.resourceserver.jwt.jwk-set-uri=http://test
 
 # By default test tries to create namespace dynamically - in this case leave them empty.
 # If in your kubernetes that is not possible, set a fixed namespace here
-integrationTest.dataJobsNamespace=${DEPLOYMENT_K8S_NAMESPACE}
-integrationTest.controlNamespace=${CONTROL_K8S_NAMESPACE}
+integrationTest.dataJobsNamespace=${DEPLOYMENT_K8S_NAMESPACE:default}
+integrationTest.controlNamespace=${CONTROL_K8S_NAMESPACE:default}
 
 datajobs.builder.image=registry.hub.docker.com/versatiledatakit/job-builder:1.2.3
-datajobs.proxy.repositoryUrl=${DOCKER_REGISTRY_URL}
```

Comment:
what about ci/cd usages, how does this change affect those?

Reply:
These were always the same value. But I have reverted to avoid confusion.